### PR TITLE
Fix PID collision in the test data - but doesn't solve e2e tests still

### PR DESCRIPTION
**Pull request by wimax-grapl**
### Which issue does this PR correspond to?
#231 and #389 

### What changes does this PR make to Grapl? Why?
Change the test data to remove a PID collision.

### How were these changes tested?
ran e2e locally; also running them on here, and keeping an eye on other people's builds in the coming days.

### CODE REVIEW

**Review by wimax-grapl — COMMENTED** [review]
oh theres some encoding issues

File: etc/sample_data/eventlog.xml [hunk 1]
```diff
@@ -392,7 +392,8 @@
...
...</Data><Data Name='Image'>C:\Program Files (x86)\Google\Update\GoogleUpdate.exe</Data><Data Name='FileVersion'>1.3.34.11</Data><Data Name='Description'>Google Installer</Data><Data Name='Product'>Google Update</Data><Data Name='Company'>Google LLC</Data><Data Name='OriginalFileName'>GoogleUpdate.exe</Data><Data Name='CommandLine'>"C:\Program Files (x86)\Google\Update\GoogleUpdate.exe" /registermsihelper</Data><Data Name='CurrentDirectory'>C:\Program Files (x86)\Google\Update\1.3.34.11\</Data><Data Name='User'>NT AUTHORITY\SYSTEM</Data><Data Name='LogonGuid'>{87E8D3BD-998D-5D38-0000-0020E7030000}</Data><Data Name='LogonId'>0x3e7</Data><Data Name='TerminalSessionId'>0</Data><Data Name='IntegrityLevel'>System</Data><Data Name='Hashes'>MD5=82F657B0AEE67A6A560321CF0927F9F7,SHA256=794CF7644115198DB451431BCA7C89FF9A97550482B1E3F7F13EB7ACA6120A11</Data><Data Name='ParentProcessGuid'>{87E8D3BD-9D3F-5D38-0000-00103C561700}</Data><Data Name='ParentProcessId'>3328</Data><Data Name='ParentImage'>C:\Program Files (x86)\Google\Update\GoogleUpdate.exe</Data><Data Name='ParentCommandLine'>"C:\Program Files (x86)\Google\Update\GoogleUpdate.exe" /ua /installsource scheduler</Data></EventData></Event>
```
Comment (inickles-grapl) [inline etc/sample_data/eventlog.xml:396]: The collision is not within this file?

**Review by colin-grapl — APPROVED** [review]

File: etc/sample_data/eventlog.xml (hunk 1 above)
Comment (wimax-grapl) [reply etc/sample_data/eventlog.xml:396]: image
it is, just lower than the fiels-changed view shows

**Comment by wimax-grapl** [comment]
As we can see, this did not solve the problem. However, hopefully by viewing the state of the db, maybe we can figure it out. I've landed a diff that enables that.

**Pull request merged**